### PR TITLE
Updated lists of failing/unsupported tests outside of Windows

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: tests/src/Interop/ReversePInvoke/Marshalling/CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,4 @@ set(SOURCES MarshalBoolArrayDLL.cpp MarshalBoolArrayDLL.def)
 add_library (MarshalBoolArrayDLL SHARED ${SOURCES})
 
 # add the install targets
-install (TARGETS MarshalBoolArrayDLL DESTINATION bin)
-
-
+install (TARGETS MarshalBoolArrayDLL DESTINATION bin)
```
Comment (adityamandaleeka) [inline tests/src/Interop/ReversePInvoke/Marshalling/CMakeLists.txt]: Looks like nothing really changed in this file. Can you just undo this change?

Comment (wtgodbe) [inline tests/src/Interop/ReversePInvoke/Marshalling/CMakeLists.txt]: It is done

**Comment by jkotas** [comment]
These broken globalization and path tests should be deleted from coreclr repo right away instead of being maintained in a list.

There is equivalent coverage in corefx for all these broken globalization and path test. For example, 

https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/IO/PathTests.cs are tests for Path. They verify order of magnitude more cases than all Path tests in CoreCLR combined, and they have all been fixed up for Unix.Moreover, the .NET Core Path implementation lives in [corefx](https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/src/System/IO/Path.cs) so the Path tests do even belong to CoreCLR.

**Comment by jkotas** [comment]
@wtgodbe  anything left good in this PR, or can this be closed?

**Comment by wtgodbe** [comment]
It can be closed

Sent from my iPhone

On Nov 20, 2015, at 6:13 PM, Jan Kotas <notifications@github.com<mailto:notifications@github.com>> wrote:


@wtgodbe<https://github.com/wtgodbe> anything left good in this PR, or can this be closed?

-
